libraries/cms/application/site.php

```diff
@@ -651,6 +621,25 @@ protected function initialiseApp($options = array())
 	}
 
 	/**
+	 * Language check
+	 *
+	 * @param  string  Language to check.
+	 *
+	 * @return  mixed  string  The passed language if it is an available language.
+	 *                 null    If the language is not an available language.
+	 *
+	 * @since   3.4.2
+	 */
+	protected function checkLang($lang)
```

Comment:
... well, maybe it may make *some* sense for en extendible class, but IMHO not really here, in a final class...
I could had also declared the method as "private"...